From eecdd7cc438df662f852478c47d4333cde983e79 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Thu, 7 Sep 2023 18:11:36 +0200
Subject: [PATCH] arm64: dts: hummingboard-t: update humidty-sensor i2c address

Humidity-sensor was not available in revisions 1.1 or earlier.
Since revision 1.2 it is available at address 0x41 - update the node
name and reg property accordingly.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts b/arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts
index f35e895bb4b6..d579f176c717 100644
--- a/arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts
+++ b/arch/arm64/boot/dts/ti/k3-am642-hummingboard-t.dts
@@ -395,10 +395,10 @@ &main_i2c0 {
 	pinctrl-0 = <&main_i2c0_pins_default>, <&main_i2c0_int_pins_default>;
 
 	/* Carrier */
-	humidity_sensor: hdc2010@40 {
+	humidity_sensor: hdc2010@41 {
 		status = "okay";
 		compatible = "ti,hdc2010";
-		reg = <0x40>;
+		reg = <0x41>;
 		interrupt-parent = <&main_gpio0>;
 		interrupts = <37 IRQ_TYPE_EDGE_FALLING>;
 	};
-- 
2.35.3

